ui/apps/platform/src/Components/PatternFly/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -1,34 +1,42 @@
 import React, { ReactElement, ReactNode } from 'react';
 import { Modal, ModalVariant, Button, ButtonVariant } from '@patternfly/react-core';
 
+import './ConfirmationModal.css';
+
 type ConfirmationModalProps = {
     ariaLabel: string;
+    title?: string;
     confirmText: string;
     onConfirm: () => void;
     onCancel: () => void;
     isOpen: boolean;
     isLoading?: boolean; // if modal remains open until finally block of request promise
+    isDestructiveAction?: boolean;
     children: ReactNode;
 };
 
 function ConfirmationModal({
     ariaLabel,
+    title,
     confirmText,
     onConfirm,
     onCancel,
     isOpen,
     isLoading,
+    isDestructiveAction = true,
     children,
 }: ConfirmationModalProps): ReactElement {
     return (
         <Modal
             isOpen={isOpen}
             variant={ModalVariant.small}
+            title={title || ''}
             actions={[
                 <Button
                     key="confirm"
-                    variant={ButtonVariant.danger}
+                    variant={isDestructiveAction ? ButtonVariant.danger : ButtonVariant.primary}
                     onClick={onConfirm}
+                    className="pf-confirmation-modal-confirm-btn"
                     isDisabled={isLoading}
```

Comment:
Does the component need a new optional prop `isConfirmDisabled = false` and confirm `Button` element prop `isDisabled={isConfirmDisabled || isLoading}` for the situation with multiple notifiers and none selected?

For something like `isConfirmDisabled={notifiers.length > 1 && selectedNotifierIds.length === 0}`